CHANGELOG.md

```diff
@@ -510,7 +510,7 @@
 * Update before oauth2.mdx ([#2299](https://github.com/ory/hydra/issues/2299)) ([d2ee4f6](https://github.com/ory/hydra/commit/d2ee4f6cd308a2b61fd4ef7f8fcebb2901190a58)), closes [#2295](https://github.com/ory/hydra/issues/2295)
 * Update npm package name ([#2302](https://github.com/ory/hydra/issues/2302)) ([d05d82e](https://github.com/ory/hydra/commit/d05d82e926a726fd4fe0179363a140ca59e40c10)):
 
-    > Changed npm client package from @oryd/hydra-client to @ory/hydra-client
+    > Changed npm client package from @ory/hydra-client to @ory/hydra-client
```

Comment:
Revert this please. Don't change the changelogs and upgrade guides retrospectively.